GitSpace/Sources/Views/Common/SuggestBlockView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SuggestBlockView.swift
+//  GitSpace
+//
+//  Created by Da Hae Lee on 2023/04/19.
+//
+
+import SwiftUI
+
+struct SuggestBlockView: View {
+    @Environment(\.dismiss) private var dismiss
+    @Binding var isBlockViewShowing: Bool
+    @Binding var isSuggestBlockViewShowing: Bool
+    
+    var body: some View {
+        VStack {
+            VStack {
+//                let _ = print("!!\(isSuggestBlockViewShowing) \(isBlockViewShowing)")
+                GSText.CustomTextView(style: .title1, string: "Report Submitted")
+                GSText.CustomTextView(style: .description, string: "Your report is submitted. ")
+            }
+            .padding()
+            
+            VStack(alignment: .leading) {
+                GSText.CustomTextView(style: .title3, string: "Well... Do you want to Block this user?")
```

Comment:
이미 Block한 유저를 신고했을 경우에,
24번 라인 이후의 코드를 분기 처리 해줘야겠네요!

- 이미 Block한 유저라면
```suggestion
                GSButton.CustomButtonView(style: .plainText(isDestructive: false)) {
                    isSuggestBlockViewShowing.toggle()
                } label: {
                    Text("Ok")
                }
                .padding([.leading, .trailing], 20)
```
- Block하지 않은 유저라면
아래의 코드 진행